edgedb-client/src/builder.rs

```diff
@@ -113,6 +114,12 @@ pub(crate) enum Address {
 
 struct DisplayAddr<'a>(Option<&'a Address>);
 
+struct DSN {
```

Comment:
We treat abbreviations as words in Rust. So:
```suggestion
struct Dsn {
```

This makes it easier to name variables and constants with that type.